index.html

```diff
@@ -0,0 +1,102 @@
+                </aside>
+            </section>
+
+            <section class="cards col-11">
+                <div class="card one">
+                    <div class="card-body">
+                    <h5 class="card-title">Server Side APIs</h5>
+                    <p class="card-text"> A server-side web API is a programmatic interface consisting of one or more publicly exposed endpoints to a defined request–response message system, typically expressed in JSON or XML, which is exposed via the web—most commonly by means of an HTTP-based web server.</p>
+                    <p> They are called server-side APIs is because the code executes on the server before it is sent to the browser. Executing code on the server is often more efficient, and also makes it easier to create web pages that work correctly on multiple browsers. </p>
+                    </div>
+                </div>
+                <hr>
+                
+                <div class="card two">
+                        <div class="card-body">
+                        <h5 class="card-title">Documentation</h5>
+                        <img src="./Assets/IMG/generic-image-placeholder.png" alt="placeholder"/>
+                        <p class="card-text"> Each API has its own documentation. On this page we will show you how to read through them and explain what key things are important to look for in these documentations. It also includes a list of free public APIs you can use.</p>
+                        </div>
+                </div>
+                <hr>
+
+                <div class="card three">
+                    <div class="card-body">
+                    <h5 class="card-title">API Demo</h5>
+                    <img src="./Assets/IMG/generic-image-placeholder.png" alt="placeholder"/>
+                    <p class="card-text">On this page we will go through the steps of getting and using data from an API. It explains and shows the coding process.</p>
+                    </div>
+                </div>
+                <hr>
+
+                <div class="card four">
+                    <div class="card-body">
+                    <h5 class="card-title">API Practice</h5>
+                    <img src="./Assets/IMG/generic-image-placeholder.png" alt="placeholder"/>
+                    <p class="card-text">On this page we will go through a tutorial for you to get involved and learn hands-on. All you have to do is be on Google Chrome browser and follow our directions.</p>
+                    </div>
+                </div>
+             </section>
+              
+              
+        </main>
+    </body>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.1/umd/popper.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/js/bootstrap.min.js"></script> 
+    <script src="./Assets/JS/script.js"></script>
```

Comment:
If you change the file and folders to lowercase, this src attribute will need to be updated